### ISSUE TITLE
Usage and importing

### ISSUE THREAD

**Issue by MightyAlex200** [timmahrt/praatIO.js#8]
As far as I can see, the documentation does not do enough to explain how a developer would import and begin using this library.
Using the CommonJS `require('praatio')` does not work. And neither does `import('praatio')` nor `import { Textgrid /*etc*/ } from 'praatio'`. I can see from the project files that this is intended to be transpiled with Babel, but the `package.json` file does not include a build script. As far as I can tell, this project is unusable from node simply by installing and importing it in any way.

**Comment by timmahrt** [comment]
Thanks for your message.  This week, I will investigate and fix it/try to add some documentation or something.

In the meantime, I've used this library in one node project.  Here is an example where it gets imported:
https://github.com/timmahrt/web_transcription_tools/blob/main/src/components/TextgridLoader.vue

I don't do that much javascript development, so if that won't work for most use cases (is it vue specific?) I could believe that.  Sorry for the headaches.

**Comment by MightyAlex200** [comment]
Not headaches! The problems must be caused by the fact that the library is made for compilation to the browser, while I am trying to use it in node. I haven't tested it thoroughly, but as far as I can tell most if not all are the issues are caused by module management. I think you can fix it for my use case without breaking it for yours by adding `"type": "module"` to your `package.json` and adding the `.js` extension to all local imports. I can create a PR for that if you'd like.

**Comment by MightyAlex200** [comment]
Yup! Seems to work for my use case. You would need to test it on your own use-case to see if it works there too but the changes are very small. I have created a pull request #9

**Comment by timmahrt** [comment]
Thanks for the PR!

Are you in a rush? For my existing (ancient) test project, I seem to be stuck in dependency hell.  I could start with a fresh project, but that could take some time too. I might have time this weekend to hash something out.

**Comment by MightyAlex200** [comment]
No rush! I'm not sure when my project will be done, and I'm using this library for only a small part of it. I'm perfectly fine importing it from my disk for now.

**Comment by timmahrt** [comment]
Nothing worked the other day and today everything worked just fine.  🤷 

I've merged your PR and bumped the version to `2.3.3`.  Thanks again for the PR!
